src/web/server/htmlTemplate.ts

```diff
@@ -221,9 +219,8 @@ export const htmlTemplate = ({
 						keywords,
 					).replace(/%20/g, '+')}" />
                 </noscript>
-                ${priorityScriptTags.join('\n')}
                 ${loadableConfigScripts.join('\n')}
-                ${loadableScripts.join('\n')}
+                ${priorityScriptTags.join('\n')}
```

Comment:
I presume it was but just in case: this change in order was intentional?

Reply:
Yeah, I removed `loadableScripts`, it's now a part of the `priorityScriptTags` array